include/utils.h

```diff
@@ -152,27 +151,49 @@ class AlignedFileReader;
 namespace diskann {
   static const size_t MAX_SIZE_OF_STREAMBUF = 2LL * 1024 * 1024 * 1024;
 
+  inline void print_error_and_terminate(std::stringstream& error_stream) {
+    diskann::cerr << error_stream.str() << std::endl;
+    throw diskann::ANNException(error_stream.str(), -1, __FUNCSIG__, __FILE__,
+                                __LINE__);
+  }
+
+  inline void report_memory_allocation_failure() {
+    std::stringstream stream;
+    stream << "Memory Allocation Failed.";
+    print_error_and_terminate(stream);
+  }
+
+  inline void report_misallignment_of_requested_size() {
+    std::stringstream stream;
+    stream << "Requested memory size is misalligned and can not be allocated.";
```

Comment:
correct typo in misalligned.
Add info on "Requsted memory size is not a multiple of {align}. Can not be allocated". Print value of align so it's easier to interpret error.

Reply:
Added the info about align in new commit.